src/crimson/net/SocketMessenger.h

```diff
@@ -36,12 +36,14 @@ class SocketMessenger final : public Messenger {
   std::set<SocketConnectionRef> accepting_conns;
   using Throttle = ceph::thread::Throttle;
   ceph::net::PolicySet<Throttle> policy_set;
+  const std::string logic_name;
```

Comment:
could you elaborate a little bit on what "logic_name" is for ? for instance, how are we supposed to use it in the context of crimson-osd?

Reply:
also i'd suggest move the addition of "logic_name" to a separated commit. it is not related to "fixes addresses during banner exchange".

Reply:
splitted to commit c5f0c44a0856a5b14ef3e8077102862c3e65a97e.

It might be helpful for debugging to distinguish messengers with meaningful names. For example, to explain the role of the messenger in the logs, one OSD has 7 messengers for different purposes.

Reply:
might want to document this variable with a comment.

Reply:
done.